src/lib/dropdowns/store_dropdown/flags/AUFlag.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const AUFlag = props => (
```

Comment:
According to this page https://blog.logrocket.com/how-to-use-svgs-in-react/ We should be able to cover these svgs a lot more simply:

`import {ReactComponent as AUFlag} from './AUFlag.svg';`

Reply:
TIL create-react-app has a built in SVG loader. Fixed in https://github.com/iFixit/product-page-components/pull/31/commits/e38cf452ac8ef05b70f5e9f99a07c2ffcf5c33be.